File: Week 6/points.py
try:
    points = int(input("how many points did you lose: "))
except ValueError:
    print("That is not a number!")
    exit()
if points < 2:
    print("You lost fewer points than me.\n")
elif points > 2:
    print("You lost more points than me.\n")
else:
    print("You lost the same number of points as me.\n")

File: Week 1/lecture_tasks/points.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
    const int POINTBREAK = 2;
    int points = get_int("How many points did you lose? ");
    if (points < POINTBREAK) {
        printf("You lost fewer points than me.\n");
    } else if (points > POINTBREAK) {
        printf("You lost more points than me.\n");
    } else {
        printf("You lost more points than me.\n");
    }
}